packages/web/src/pages/App.js

```diff
@@ -514,11 +516,20 @@ class App extends Component {
                 </Route>
                 <Route exact path={SIGN_UP_PAGE} isMobile={isMobileClient}>
                   {isSignInRedesignEnabled ? (
-                    <SignUpRootPage />
+                    <Redirect to={SIGN_UP_START_PAGE} />
```

Comment:
Keeps all the links the same (i.e. `/signup` will redirect to `/signup/create-account`)